modules/shared/components/molecules/PostViewHeader/UserInfo/PostViewHeader.stories.tsx

```diff
@@ -34,6 +34,6 @@ export const postViewHeader = Template.bind({});
 postViewHeader.args = {
   size: 'medium',
   variant: 'filled',
-  name: 'Marzouk el akta3',
```

Comment:
It was nice. I didn't mean to change it. I like writing funny things in the tests. please keep it 😂